### PR TITLE
#3256 Provider Scaling out Interface-Initialize netty related classes

### LINKED ISSUE #3256: Provider Scaling out Interface
## Sequence diagram

sequence diagram

## Feature
- [x] Create/Start Scaling job interface
- [x] Query Scaling job interface
    - [x] List all Scaling job interface
    - [x] Show scaling progress and detail #3617 
- [x] Stop/Delete Scaling job interface

### PR DESCRIPTION
**Pull request by ssxlulu**
For #3256.

Changes proposed in this pull request:
- Initialize netty related classes

### CODE REVIEW

File: sharding-scaling/sharding-scaling-core/src/main/java/org/apache/shardingsphere/shardingscaling/core/web/HttpServerHandler.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+/**
+ * Http server handler.
+ *
+ * @author ssxlulu
+ */
+@Slf4j
+public final class HttpServerHandler extends SimpleChannelInboundHandler<FullHttpRequest> {
+
+    private static final Pattern URL_PATTERN = Pattern.compile("(^/shardingscaling/start)|(^/shardingscaling/(progress|stop)/\\d+)",
```
Comment (KomachiSion) [inline sharding-scaling/sharding-scaling-core/src/main/java/org/apache/shardingsphere/shardingscaling/core/web/HttpServerHandler.java]: Is `queryProgress` better than `progress`?

File: sharding-scaling/sharding-scaling-core/src/main/java/org/apache/shardingsphere/shardingscaling/core/web/HttpServerHandler.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            startShardingScalingJob(requestBody);
+            response("start", channelHandlerContext, HttpResponseStatus.OK, request);
+            return;
+        }
+        if (requestPath.contains("/shardingscaling/progress/") && method.equals(HttpMethod.GET)) {
+            //TODO
+            response("progress", channelHandlerContext, HttpResponseStatus.OK, request);
+            return;
+        }
+        if (requestPath.contains("/shardingscaling/stop/") && method.equals(HttpMethod.DELETE)) {
```
Comment (KomachiSion) [inline sharding-scaling/sharding-scaling-core/src/main/java/org/apache/shardingsphere/shardingscaling/core/web/HttpServerHandler.java]: `stop` interface should be not DELETE method

**Comment by ssxlulu** [comment]
> Can you change the `Fixes #issue` to `For #issue`, I think this PR can't fix issue completely.

OK.

File: sharding-scaling/sharding-scaling-core/src/main/java/org/apache/shardingsphere/shardingscaling/core/web/HttpServerHandler.java (hunk 2 above)
Comment (ssxlulu) [reply sharding-scaling/sharding-scaling-core/src/main/java/org/apache/shardingsphere/shardingscaling/core/web/HttpServerHandler.java]: Yes, the APIS should be changed, here are the final discussion result:
```
post:
job/start/
job/stop/

get:
job/progress/${id}
```

**Pull request merged**